### PR TITLE
Configure Logging

### PR DESCRIPTION
**Pull request by masterashu**
closes: #7

### CODE REVIEW

File: mailgun/mail.py [hunk 1]
```diff
@@ -57,17 +65,21 @@ def send_mass_mail(datatuples, fail_silently=True):
...
+            try:
+                response = my_session.post(
+                    settings.MAILGUN_BASE_URL,
+                    data={'from': sender,
+                          'to': to,
+                          'subject': subject,
+                          'text': body})
+            except:
+                logger.error('Unable to Send Post Request')
                 continue
```
Comment (AnubhavUjjawal) [inline mailgun/mail.py]: Remove `continue`

File: mailgun/mail.py [hunk 2]
```diff
@@ -11,23 +15,27 @@ def send_mail(subject, body, to, sender=settings.EMAIL_HOST_USER, fail_silently=
...
+                  'text': body
+                  })
+    except:
+        logger.error('Unable to Send Post Request')
+        return None
     # If Response is not OK(200) show failed message
     if not fail_silently and response.status_code != 200:
-        # TODO raise any error or log failed email.
-        pass
+        print(__name__)
```
Comment (AnubhavUjjawal) [inline mailgun/mail.py]: Remove `print`

File: .env [hunk 3]
```diff
@@ -11,3 +11,6 @@ EMAIL_HOST_USER=mail_user
 EMAIL_HOST_PASSWORD=password
 EMAIL_USE_TLS=True
 DEFAULT_FROM_EMAIL=donotreply@example.com
+U_LOGFILE_NAME=log_data
+U_LOGFILE_SIZE=1048576
+U_LOGFILE_COUNT=12
```
Comment (AnubhavUjjawal) [inline .env]: Where are these 3 being used ?

File: IIITCouncil/settings.py [hunk 4]
```diff
@@ -134,6 +134,53 @@
...
+        },
+    },
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+        },
+        'logfile': {
+            'level': 'DEBUG',
+            'class': 'logging.handlers.RotatingFileHandler',
+            'filename': './logs/MAILGUN.log',
```
Comment (AnubhavUjjawal) [inline IIITCouncil/settings.py]: Configurable from env variables ?

File: mailgun/mail.py (hunk 1 above)
Comment (masterashu) [reply mailgun/mail.py]: Done

File: mailgun/mail.py (hunk 2 above)
Comment (masterashu) [reply mailgun/mail.py]: Done

File: .env (hunk 3 above)
Comment (masterashu) [reply .env]: Okay.

File: IIITCouncil/settings.py (hunk 4 above)
Comment (masterashu) [reply IIITCouncil/settings.py]: Removed env variables. All the properties are now mentioned in settings.py

**Pull request merged**